files/k8s-resource/calico-crd-installer.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
```

Comment:
These RBAC resources will be left after the job runs. I don't think this is problematic.